pod-master/podmaster.go

```diff
@@ -159,6 +165,10 @@ func validateFlags(c *config) {
 	if len(c.dest) == 0 {
 		glog.Fatalf("--dest-file=<some-file> is required")
 	}
+	//either all are populated, or none are populated
```

Comment:
please move this into the validateFlags function (the ability to return early will clean up this condition)